code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -14,9 +14,11 @@
 	melee_damage_upper = 30
 	melee_damage_type = RED_DAMAGE
 	armortype = RED_DAMAGE
+	blood_volume = 0
 	rapid_melee = 2
 	stat_attack = DEAD
 	ranged = TRUE
+	density = FALSE
```

Comment:
Look, this will make it so bullets won't hit it. As funny as it is, you should come up with a different solution.